Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -630,7 +630,9 @@ - (void)lt_cleanup {
             __autoreleasing NSError *error;
             BOOL synchronized = [_currentLogFileHandle synchronizeAndReturnError:&error];
             if (!synchronized) {
-                NSLogError(@"DDFileLogger: Failed to synchronize file: %@", error);
+                if (error) {
```

Comment:
Inlining this in above's `if (!synchronized)` to become `if (!synchronized && error)` would reduce nesting.